cairo/api.go

```diff
@@ -32,6 +33,22 @@ func NewFromJunoBlockID(id rpc.BlockID) BlockID {
 	}
 }
 
+type LatestBlockResponse struct {
+	Block *types.CompactBlock `json:"latest_block"`
+	Err   *jsonrpc.Error      `json:"err"`
+}
+
+// 不是很确定这里是不是其实不需要request的内容，直接response
```

Comment:
Pls use English to write comments and you use `fixme: ` as the beginning of the comment if you are not sure.

Reply:
Fixed in next commit，thanks :)